packages/xeus-extension/schema/xeus-kernel-status.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "Xeus Kernels Settings",
+  "description": "Xeus Kernels Settings.",
+  "jupyter.lab.toolbars": {
+    "Notebook": [{ "name": "xeusKernelLogs", "rank": 3000 }]
```

Comment:
How about placing it to the right of the kernel status item, instead of to the right of the scrollbar button?

![image](https://github.com/user-attachments/assets/82c77f0e-bcc1-4889-8054-555457c306fe)